test/e2e/libpod_suite_remote_test.go

```diff
@@ -24,16 +24,10 @@ func IsRemote() bool {
 }
 
 func SkipIfRemote(reason string) {
-	ginkgo.Skip("[remote]: " + reason)
-}
-
-func SkipIfRootlessCgroupsV1() {
-}
-
-func SkipIfRootless() {
-	if os.Geteuid() != 0 {
-		ginkgo.Skip("This function is not enabled for rootless podman")
+	if reason == "" {
+		panic("SkiptIfRemote must specify a reason to skip")
```

Comment:
One more Skipt-with-a-t

Reply:
I am sure we will need another pass, since I can not tests the tests locally right now.